include/lsst/meas/base/PixelFlags.h

```diff
@@ -94,10 +110,12 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
     ) const;
 
 private:
-
+    template <typename MaskedImageT>
+    class FootprintBits;
+    typedef afw::image::MaskedImage<float> MaskedImageT;
+    void updateFlags(const KeyMap *,const FootprintBits<MaskedImageT> *,afw::table::SourceRecord & ) const;
```

Comment:
@RobertLuptonTheGood could you create an RFC on that?  I'm not opposed, but I don't like deciding coding practices piecemeal by code review.